backend/src/v4/routes/maps.js

```diff
@@ -546,39 +494,26 @@ function requestMapTile(req, res, domain, uri) {
 	});
 }
 
-function requestHereMapTile(req, res, domain, uri) {
-	uri += "?apiKey=" + config.here.apiKey;
+function requestHereMapTile(req, res, domain, resource, style, features) {
+	const featureList = features ? features.split(",") : [];
 
-	if (req.query.congestion) {
-		uri += "&congestion=" + req.query.congestion;
-	}
-	if (req.query.lg) {
-		uri += "&lg=" + req.query.lg;
-	}
-	if (req.query.lg2) {
-		uri += "&lg2=" + req.query.lg2;
-	}
-	if (req.query.min_traffic_congestion) {
-		uri += "&min_traffic_congestion=" + req.query.min_traffic_congestion;
-	}
-	if (req.query.pois) {
-		uri += "&pois=" + req.query.pois;
-	}
-	if (req.query.ppi) {
-		uri += "&ppi=" + req.query.ppi;
-	}
-	if (req.query.pview) {
-		uri += "&pview=" + req.query.pview;
+	let uri = `/v3/${resource}/mc/${req.params.zoomLevel}/${req.params.gridx}/${req.params.gridy}/png8?apiKey=${config.here.apiKey}`;
+
+	if (req.query.features) {
```

Comment:
do we expect this query string from unity?

it doesn't look like you had it before, so it sounds like a no unless i'm missing something?

If Unity never calls this endpoint with it, don't bother supporting it.

Reply:
Same as for styles. We don't send any query string from Unity for features, but you could potentially have multiple features enabled e.g. POIs, congestion zones, environmental zones. At the moment the Here styles/features are mainly presented as different maps in the IO drop down list. Technically, it is possible to have some of these features mimic the overlays we have for Congestion and Truck Restrictions but this would require changes on Unity and possibly IO frontend.

Removed since we don't use it.